src/source/Include_i.h

```diff
@@ -149,11 +151,16 @@ STATUS generateJSONSafeString(PCHAR, UINT32);
 #include "Rtp/Codecs/RtpH264Payloader.h"
 #include "Rtp/Codecs/RtpOpusPayloader.h"
 #include "Rtp/Codecs/RtpG711Payloader.h"
+#include <poll.h>
```

Comment:
Where is this being used? Do you need it?

Reply:
Yes, it doesn't compile without this change, as things like `pollfd` won't be defined in the include chain (when `-DNO_SIGNALING_SERVICE` is set. I assume this is something that is currently included by libwebsockets. I want to check about cross-platform concerns here (if libwebsockets simply included `poll.h` or if there is cross-platform polyfil)

Reply:
To help me out, can I assume the CI will check for me if this is an issue for supported platforms? Or are there are some manual checks I may need to do for things like windows support (is windows supported?)?

Reply:
> To help me out, can I assume the CI will check for me if this is an issue for supported platforms? Or are there are some manual checks I may need to do for things like windows support (is windows supported?)?

Your build seem to have passed the checks which means that the target platforms in Travis CI have build and run OK.

@disa6302 could you comment on this please?

Reply:
@kev-the-dev , we still do support Windows. The reason the test is disabled is because some manual steps did not work in CI. With the manual steps for Windows highlighted [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#cmake-arguments), we would want this patch to work on Windows as well.